examples/thermomechanics/inputs/thermal_crack.json

```diff
@@ -1,16 +1,16 @@
 {
-    "num_cells"             : {"value": [501, 101, 11]},
-    "system_size"           : {"value": [0.05, 0.01, 0.001], "unit": "m"},
-    "density"               : {"value": 3980, "unit": "kg/m^3"},
-    "elastic_modulus"       : {"value": 370e+9, "unit": "Pa"},
-    "fracture_energy"       : {"value": 24.32, "unit": "J/m^2"},
-    "thermal_coefficient"   : {"value": 7.5E-6, "unit": "oC^{-1}"},
-    "reference_temperature" : {"value": 300.0, "unit": "oC"},
-    "water_temperature"     : {"value": 20.0, "unit": "oC"},
-    "pulse_ramping_time"    : {"value": 0.001, "unit": "s"},
-    "horizon"               : {"value": 3.0e-4, "unit": "m"},  
-    "final_time"            : {"value": 0.002, "unit": "s"},
-    "timestep"              : {"value": 1.5E-8, "unit": "s"},
-    "output_frequency"      : {"value": 500},
-    "output_reference"      : {"value": true}
+    "num_cells"                     : {"value": [501, 101, 11]},
+    "system_size"                   : {"value": [0.05, 0.01, 0.001], "unit": "m"},
+    "density"                       : {"value": 3980, "unit": "kg/m^3"},
+    "elastic_modulus"               : {"value": 370e+9, "unit": "Pa"},
+    "fracture_energy"               : {"value": 24.32, "unit": "J/m^2"},
+    "thermal_coefficient"           : {"value": 7.5E-6, "unit": "oC^{-1}"},
+    "reference_temperature"         : {"value": 300.0, "unit": "oC"},
+    "water_bath_temperature"        : {"value": 20.0, "unit": "oC"},
```

Comment:
I would rather `surface_temperature` - this seems too specific

Reply:
Would "bath_temperature" be OK? Or would you prefer something even more general?

Reply:
Updated